test/number/div.wren

```diff
@@ -0,0 +1,5 @@
+IO.print((8).div(2))   // expect: 4
+IO.print((8.5).div(2)) // expect: 4
+IO.print((9).div(2))   // expect: 4
+IO.print((9.9).div(2)) // expect: 4
+IO.print((10).div(2))  // expect: 5
```

Comment:
More stuff to test:

* Large number
* Zero LHS.
* Zero RHS.
* Floating point RHS.
* Negative arguments.